.changeset/good-lemons-lick.md

```diff
@@ -0,0 +1,14 @@
+---
+'@backstage/plugin-scaffolder-backend-module-confluence-to-markdown': patch
+'@backstage/plugin-scaffolder-backend-module-cookiecutter': patch
+'@backstage/plugin-scaffolder-backend-module-bitbucket': patch
```

Comment:
just a note:
I plan to create a PR to split this into "bitbucket-cloud" and "bitbucket-server" similar to other packages.

Reply:
@pjungermann nice - makes sense! Please do! :pray: